per-seat-subscriptions/server/go/README.md

```diff
@@ -0,0 +1,18 @@
+# Subscriptions with fixed price
```

Comment:
And update here? (I can take these in a separate PR after we merge this one, too, if you prefer -- I've got a PR in flight to update other readmes, can take the string in `package.json` at the same time if it would help. Probably won't ship that one until next week though.

Reply:
for now just removed the go dir until we get it updated